heareval/tasks/pipeline.py

```diff
@@ -575,18 +689,17 @@ class FinalizeCorpus(WorkTask):
 
     def requires(self):
         # Will copy the resampled data and the traintestmeta and the vocabmeta
-        splits = [p["name"] for p in self.data_config["splits"]]
         return {
-            "resample": [
-                ResampleSubCorpus(
-                    sr=sr,
-                    split=split,
-                    metadata=self.metadata,
-                    data_config=self.data_config,
-                )
-                for sr in self.sample_rates
-                for split in splits
-            ],
+            "stats": GenerateAudioStats(
+                metadata=self.metadata,
+                sample_rates=self.sample_rates,
+                data_config=self.data_config,
+            ),
+            "resample": ResampleSubcorpuses(
```

Comment:
Why have ResampleSubcorpuses a requirement here as well? It is already in the GenerateAudioStats so these will all necessarily be completed by the time we get here.

Reply:
Because a) we might remove the GenerateAudioStats. b) We still require the workdirectory of the ResampleSubcorpuses to copy the final outputs